src/i18n/en.json

```diff
@@ -86,10 +86,9 @@
   "importKey": {
     "tooltip": "Please provide a valid Reward Address to proceed.",
     "pageTitle": "Import Reward Address",
-    "rewardAddress": "Please enter your Reward Address in SS58 format (it should start with the letters \"st\")",
+    "rewardAddress": "Please enter your Reward Address in SS58 format!",
```

Comment:
I think most ppl don't know what is SS58, so I would keep `(it should start with the letters \"st\")`, alternatively we could add a link somewhere

Reply:
It does not have to start with `st` though, any valid `SS58` would do, that's why I wanted to get rid of `st`. 
Maybe we can just say it's in the wrong format, but then the user wouldn't know which format to follow and how to fix the invalid form. 

Maybe saying `it should be in "st" format` will be okay although it is not necessary. It should be a clear tip to the user.
I want to get @nazar-pc 's opinion on that though

Reply:
We already told the community that any address should work and I feel like we should stick to that until we hit mainnet at least.

I also don't think we need to mention SS58 in any way to the end user, we should just say something like "proper wallet address from Subwallet or Polkadot.js extension" (users that use Talisman or something will know it is still the same address and is compatible),

Reply:
> "proper wallet address from Subwallet or Polkadot.js extension"
 
we can go with it if that's ok with you @isSerge

Reply:
Totally